cheri-hybrid-c-guide.tex

```diff
@@ -1061,6 +1063,11 @@ \subsection{Hybrid language runtimes}
 also no requirement for the JIT or interpreter to use the same reference
 representation as the C/C++ part of a VM (though in practice it probably will).}
 
+\raarmnote{Seems useful to have a semi-hybrid mode which requires all executable
+pointers to be capabilities with executable permissions cleared in the DDC. This
+isn't compatible with code taking/storing callbacks unless explicitly updated.
+Not sure yet where would be the right section for this note.}
```

Comment:
I like that idea. It could be a useful stepping-stone between hybrid and purecap.